pallets/loans-ref/src/lib.rs

```diff
@@ -33,6 +33,13 @@
 //! | [`Pallet::update_write_off_policy()`]    | PoolAdmin |
 //! | [`Pallet::update_portfolio_valuation()`] | Any       |
 //!
+//! The following actions are performed based on changes:
+//!
+//! | Extrinsics                     | Role      |
+//! |--------------------------------|-----------|
+//! | [`Pallet::propose_change()`]   | LoanAdmin |
+//! | [`Pallet::apply_change()`]     | Borrower  |
```

Comment:
Why the differing roles for this btw? Not sure it's wrong, just curious about the choice. Maybe `apply_change` could even be permissionless..?